src/providers/WorkflowCore.Providers.Azure/Interface/ICosmosDbClient.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.Azure.Cosmos;
+
+namespace WorkflowCore.Providers.Azure.Interface
+{
+    public interface ICosmosDbClient
```

Comment:
Could we name this `ICosmosClientFactory`?

Reply:
Great. Like ICosmosClientFactory